.github/workflows/ci.yml

```diff
@@ -1,17 +1,16 @@
+---
 name: Continuous Integration
 
 on:
   push:
     branches:
       - master
-      - cmake-3.12
-      - cmake-3.8
+      - develop
 
   pull_request:
     branches:
       - master
-      - cmake-3.12
-      - cmake-3.8
+      - develop
```

Comment:
This repository doesn't have a `develop` branch. There is nothing to develop in fact. This repository is "done" as there is nothing meaningful to add to achieve its purpose. I didn't archive it only becuase CMake might further simplify some things in the future which I will integrate into this repository, but that boils down to just creating a new named branch for the current `master` (CMake 3.14) and adjusting a few lines.